### PR TITLE
Autotests gs pipelines storages

### PR DESCRIPTION
**Pull request by maryvictol**
This PR provides implementation last 7 Global Search Tests

### CODE REVIEW

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/ClusterNodeTest.java [hunk 1]
```diff
@@ -102,7 +102,7 @@ public void shouldFilterNodeByAddress() throws InterruptedException {
         final String ip = clusterMenuAO.getNodeAddress(getRunId());
         final int nodesCountBeforeFiltering = clusterMenuAO.getNodesCount();
 
-        clusterMenuAO.filerBy(ClusterMenuAO.HeaderColumn.ADDRESS, ip);
+        clusterMenuAO.filerBy(ClusterMenuAO.HeaderColumn.ADDRESS, ip.substring(0,ip.indexOf("\n")));
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/ClusterNodeTest.java]: Add a space between 0, and ip.indexOf("\n"), please

Comment (kamyshova) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/ClusterNodeTest.java]: Licence header of file should be updated according to changes

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/GlobalSearchTest.java [hunk 2]
```diff
@@ -61,13 +62,19 @@
     private final String storageFolder = "globalSearchStorageFolder" + Utils.randomSuffix();
     private final String storageFile = "globalSearchStorageFile" + Utils.randomSuffix();
     private final String storageFileContent = "globalSearchStorageFileContent" + Utils.randomSuffix();
+    private final String storageFile2 = "globalSearchStorageFile2" + Utils.randomSuffix();
     private final String storageAlias = "globalSearchStorageAlias" + Utils.randomSuffix();
-    private final String customConfigurationProfile = "custom-profile";
+    private final String toolEndpoint = "e2e-endpoints";
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/GlobalSearchTest.java]: This value should be taken from default constants

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/GlobalSearchTest.java [hunk 3]
```diff
@@ -61,13 +62,19 @@
...
+    private final String toolVersion = "latest";
+    private final String toolShortDescription = "Docker image used in E2E automated tests to verify HTTP endpoints and SSH access";
+    private final String toolDescription = "HTTP endpoints work correctly (will perform JWT authentication and print user's name)";
+    private final String customConfigurationProfile = "customprofile" + Utils.randomSuffix();
     private final String customDisk = "22";
     private final String defaultInstanceType = C.DEFAULT_INSTANCE;
     private final String defaultRegistry = C.DEFAULT_REGISTRY;
     private final String defaultGroup = C.DEFAULT_GROUP;
     private final String testingTool = C.TESTING_TOOL_NAME;
+    private final String toolWithGroup = String.format("%s/%s", C.DEFAULT_GROUP, toolEndpoint);
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/GlobalSearchTest.java]: The same as above

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/GlobalSearchTest.java [hunk 4]
```diff
@@ -299,12 +306,10 @@ public void searchForPipelineRunByID() {
     @Test
     @TestCase(value = {"EPMCMBIBPC-2660"})
     public void searchForStorage() {
-        home().sleep(2, SECONDS);
         search()
                 .search(storage)
                 .enter()
                 .sleep(2, SECONDS)
-                .validateCountSearchResults(2)
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/GlobalSearchTest.java:307]: This check is not needed anymore?

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/ToolPageAO.java [hunk 5]
```diff
@@ -53,10 +62,16 @@ public ToolPageAO validateEndpointPage() {
...
         return endpoint;
     }
-}
+
+    public void closeTab(){
+        List<String> tabs = new ArrayList<String>(getWebDriver().getWindowHandles());
+        getWebDriver().close();
+        switchTo().window(tabs.get(0));
+    }
+}
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/ToolPageAO.java]: Add a new line, please

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/Configuration.java [hunk 6]
```diff
@@ -327,4 +328,8 @@ public static Runnable confirmConfigurationChange() {
         return () -> new ConfirmationPopupAO<>(new Object()).ensureTitleIs("Are you sure you want to change configuration?").ok();
     }
 
+    public Configuration assertPageTitleIs(final String expectedTitle) {
+        assertTrue($(title()).toString().contains(expectedTitle));
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/Configuration.java]: It would be better get $(title()).getText()

Comment (kamyshova) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/Configuration.java]: Licence header of file should be updated according to changes

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/ClusterNodeTest.java (hunk 1 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/ClusterNodeTest.java]: Done

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/Configuration.java (hunk 6 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/Configuration.java]: Done

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/GlobalSearchTest.java (hunk 3 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/GlobalSearchTest.java]: Done

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/GlobalSearchTest.java (hunk 2 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/GlobalSearchTest.java]: Done

**Review by kamyshova — APPROVED** [review]

**Pull request merged**